src/Components/Web/src/Virtualization/ItemsProviderDelegate.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.Web
```

Comment:
Although we definitely want `Virtualize` itself in this namespace, I'd consider it a more open question whether we want related types like `ItemsProviderDelegate` in that same namespace, or whether there should be a `.Virtualization` namespace for those. That's because the name `ItemsProviderDelegate` is pretty generic and it's not obvious that it's specific to virtualization. Alternatively we could consider renaming it to `VirtualizationItemsProvider` or similar.

Let's not delay this PR on that question though. We can think about it during API review and do a small follow-up update if necessary.